command/recorderd/job_manager.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"sync"
+	"time"
+
+	"github.com/bitmark-inc/logger"
+)
+
+type JobManager interface {
+	Start()
+}
+
+const (
+	JobManagerPrefix = "job-manager"
+)
+
+type JobManagerChannel struct {
+	rescheduleChannel chan struct{}
+	startEventChannel chan struct{}
+	stopEventChannel  chan struct{}
+}
+
+type JobManagerData struct {
+	calendar    JobCalendar
```

Comment:
Is it possible that the job manager doesn't have to rely on the job calendar?

Reply:
I guess not in current way, manager ask calendar for next event, and based on that value to schedule next event.